spec/requests/api/providers_spec.rb

```diff
@@ -51,6 +51,36 @@
       "ipaddress" => "100.200.300.3",
     }
   end
+  let(:sample_openshift_mep) do
```

Comment:
Prefer verbosity here - I assume this stands for `sample_openshift_multi_end_point`?